Week1/work.py

```diff
@@ -66,8 +63,16 @@ def even_sort(arr):
         ## This is any even number is smaller than any odd number
     '''
 
-    ## Code Here
-    return None
+    arr.sort()
+    e=[]
+    b=[]
+    for i in range(len(arr)):
+        if arr[i]%2==0:
+            e=np.append(e,arr[i])
+        else:
+            b=np.append(b,arr[i])
+    
+    return np.concatenate((e,b))
```

Comment:
Though this is not fair on my side to tell, but do look into the expected type of output we have asked from the function and that you have used so -please use type casting and it should be fixed.

Reply:
Okay bhaiyya I inserted the typecasting and committed the changes and now it checks all. Thank you and sorry for the inconvenience.